packages/owl-bot/src/core.ts

```diff
@@ -379,7 +379,9 @@ export async function getFilesModifiedBySha(path: string, sha: string) {
     // There will be a blank line between the commit message and the
     // files list, we use this as a stopping point:
     if (file === '') break;
-    files.push(file);
+    // TODO(bcoe): discuss with rennie about what we should do with leading
+    // strict regex that didn't catch leading "/".
+    files.push(file.charAt(0) === '/' ? file : `/${file}`);
```

Comment:
@SurferJeffAtGoogle noticed this bug too, any reason not to prepend it at the outset?

Reply:
Where is the outset?

Reply:
@SurferJeffAtGoogle was thinking in the helper itself we could add the `/` prefix if it's missing?